src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,31 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            throw new DataProcessingException("Cannot add actor to DB" + actor,e);
+
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return actor;
     }
 
     @Override
     public Optional<Actor> get(Long id) {
-        return null;
+
+        try (Session session = factory.openSession()) {
+            return Optional.ofNullable(session.get(Actor.class,id));
+        } catch (Exception e) {
+            throw new DataProcessingException("Cannot find Actor from DB" + id,e);
```

Comment:
```suggestion
            throw new DataProcessingException("Cannot find Actor from DB by id: " + id,e);
```
fix similar exception messages

Reply:
not fixed